Stopped building after changes to FindEGL.cmake

The build fails with:
```
$ PKG_CONFIG_PATH=/usr/local/lib/pkgconfig cmake .. -DCMAKE_BUILD_TYPE=Release
CMake Error at cmake/FindEGL.cmake:53 (target_link_libraries):
  Cannot specify link libraries for target "GL::egl" which is not built by
  this project.
Call Stack (most recent call first):
  CMakeLists.txt:43 (find_package)


-- Configuring incomplete, errors occurred!
```

Reverting following commits fixes the problem.

fa6e47b95f3da3ae31971c84806ffcf8997d5fa6
943f85fa27da297c86f10307050fe9c790d417d0
ece5827693bf68c99555b3aa467b1f4dbfa63b74

--- a/cmake/FindEGL.cmake
+++ b/cmake/FindEGL.cmake
@@ -51,7 +51,8 @@ mark_as_advanced(EGL_INCLUDE_DIR EGL_LIBRARY)
 if (EGL_LIBRARIES AND NOT TARGET GL::egl)
     add_library(GL::egl INTERFACE IMPORTED)
     if (TARGET PkgConfig::EGL)
-        target_link_libraries(GL::egl INTERFACE PkgConfig::EGL)
+        set_property(TARGET GL::egl PROPERTY
+            INTERFACE_LINK_LIBRARIES PkgConfig::EGL)
     else ()
         set_property(TARGET GL::egl PROPERTY
             INTERFACE_LINK_LIBRARIES ${EGL_LIBRARY})

--- a/cmake/FindWPE.cmake
+++ b/cmake/FindWPE.cmake
@@ -48,7 +48,8 @@ mark_as_advanced(WPE_INCLUDE_DIR WPE_LIBRARY)
 if (WPE_LIBRARY AND NOT TARGET WPE::libwpe)
     add_library(WPE::libwpe INTERFACE IMPORTED)
     if (TARGET PkgConfig::WPE)
-        target_link_libraries(WPE::libwpe INTERFACE PkgConfig::WPE)
+        set_property(TARGET WPE::libwpe PROPERTY
+            INTERFACE_LINK_LIBRARIES PkgConfig::WPE)
     else ()
         set_property(TARGET WPE::libwpe PROPERTY
             INTERFACE_LINK_LIBRARIES "${WPE_LIBRARY}")

